app/src/main/java/org/simple/clinic/widgets/Views.kt

```diff
@@ -31,11 +31,13 @@ import timber.log.Timber
 import java.time.Duration
 
 fun EditText.showKeyboard() {
-  postDelayed({
-    this.requestFocus()
+  fun showKeyboard() {
```

Comment:
Can we rename this to something else? Seems weird to have a local function named the same as the enclosing function.

Reply:
Yep, renaming it to `openKeyboard`